ui/apps/platform/src/Containers/SystemHealth/Components/DeclarativeConfigurationHeathCard.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState, useEffect, ReactElement } from 'react';
```

Comment:
File is called `DeclarativeConfigurationHeathCard`, not `DeclarativeConfigurationHealthCard`